source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,274 @@
+.. _cpp-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, text search, efficiency
+
+Overview
+--------
+
+The MongoDB Atlas Search feature enables you to perform full-text
+searches on collections hosted on Atlas. Before you can perform Atlas
+Search queries, you must create indexes that specify which
+fields to index and how they are indexed.
+
+To learn more about Atlas Search, see the :atlas:`Atlas Search Overview
+</atlas-search/atlas-search-overview/>`.
+
+You can use the following methods on a ``mongocxx::search_index_view`` instance
+to manage your Atlas Search indexes:
+
+- ``create_one()``
+- ``create_many()``
+- ``list()``
+- ``update_one()``
+- ``drop_one()``
+
+To get a ``search_index_view`` to your collection, 
+call the ``search_indexes()`` method on your ``mongocxx::collection`` instance.
+
+.. note:: Atlas Search Index Management is Asynchronous
+
+   The {+driver-long+} manages Atlas Search indexes asynchronously. The
+   library methods described in the following sections return the server
+   response immediately, but the changes to your Search indexes take
+   place in the background and might not complete until some time later.
+
+The following sections provide code examples that demonstrate how to use
+each Atlas Search index management method.
+
+.. _cpp-atlas-search-index-create:
+
+Create a Search Index
+---------------------
+
+To create a single Atlas Search Index on a collection, call the ``create_one()`` method on a ``mongocxx::search_index_view`` instance and pass in a ``mongoxcc::search_index_model`` 
+instance specifying the Search index you want to create. 
+
+A ``mongocxx::search_index_model`` instance requires the following fields:
+
+- ``name``, a string specifying the name for your search index
+- ``definition``, a document containing the ``mappings`` field, which specifies how how to configure fields in your search index
+
+.. tip:: 
+
+   For a full list of fields you can configure in an Atlas Search index, see the :atlas:`Review Atlas Search Index Syntax </atlas-search/index-definitions/>` guide in the Atlas Search Documentation.
+
+After you create a Search index, you can perform Atlas Search queries on
```

Comment:
I decided to remove this line because the Overview section of the page mentions Atlas Search queries and links to the Atlas Overview, which readers can use to find a tutorial for creating queries.

Do you think this line is useful? I originally included it because it is in the [PHP Atlas Search Index](https://www.mongodb.com/docs/php-library/current/indexes/atlas-search-index/) page.

Reply:
fine to leave out